### ISSUE TITLE
Adding background images to the Hydrogen theme sections (Wordpress)

### ISSUE THREAD

**Issue by Cicorione** [gantry/gantry5#1363]
Hi Team,

before let me say your job is great, thanks!

I am studying the Hydrogen theme and I would like to add some pictures background to some sections for creating a basic parallax effect however I went across many css files across "../gantry5/engines/nucleus/" as well as "../themes/g5_hydrogen/scss" but I didn't find the scss which rules these particles, probably because the whole gantry framework is so huge that I have not idea where searching.

Please could you say me where I can find the classes that handle the frontend?

Thanks!

**Comment by JoomFX** [comment]
Hi @Cicorione 

By default, in Hydrogen you have a "Background Image" option for the "Showcase" section.
Just have a look at how it was done in the following files and then do the same for the sections you want:

1. `/g5_hydrogen/blueprints/styles/showcase.yaml` - here you define the fields that will show in the Admin.
2. `/g5_hydrogen/scss/hydrogen/_showcase.scss` - here you do the actual CSS/SCSS stuff.
3. `/g5_hydrogen/scss/configuration/_colors.scss` - here you define the variable default. In SCSS each variable should have a `!default` value (see line 24)

Basically, that's all.
Just play with it :wink:

**Comment by Cicorione** [comment]
Thank you sir, I'll take a look to these files!

2016-06-15 9:13 GMT-04:00 Ivo Valkov <notifications@github.com>:

**Comment by w00fz** [comment]
@Cicorione Were you able to solve this with Ivo's help?

**Comment by Cicorione** [comment]
Not I wasn't. I am still trying to figure out where I am making mistake.

I am not very prepared about SASS, basically I found these rules in the _showcase.scss:

`// Showcase
$showcase-background: #354D59 !default;
$showcase-image: false !default;
$showcase-text-color: #ffffff !default;`

And I tried to override the background with a file called _custom.scss in the custom folder

`// override showcase background image

$showcase-image: url(gantry-theme://images/stellar.jpg; no-repeat center top) !default;`

It doesn't work and I haven't found a solution yet.

Thanks

**Comment by JoomFX** [comment]
@Cicorione This is the easiest way:

1. Add the following code to your [custom.scss](http://docs.gantry.org/gantry5/tutorials/adding-a-custom-style-sheet) file:

```
#g-subfeature {
	background-image: url(gantry-theme://images/stellar.jpg);
	background-repeat: no-repeat;
	background-position: center top;
}
```

The above code will set the background properties for the "Subfeatures" section.
For the "Showcase" section you can either use the field that you already have in the Gantry 5 Admin -> Styles tab or you can use the code below. You can of course add any other background CSS property like `background-attachment`, `background-size`, etc:

```
#g-showcase {
	background-image: url(gantry-theme://images/stellar.jpg);
	background-repeat: no-repeat;
	background-position: center top;
}
```

2. Copy the `stellar.jpg` file to `../g5_hydrogen/images`

3. "Recompile CSS" in the "Base" outline so your `custom.scss` file is picked up.

You can also write pure CSS in the `custom.scss` file, so you do not need to know SCSS.

_The above is tested and works just fine. If you cannot make it work, just double-check the above steps._

**Comment by Cicorione** [comment]
Thank you very much, I wouldn't ask more because I recognize my gaps so I have been studying to know more about it.

**Comment by JoomFX** [comment]
There's no problem to ask, we are a community and we help each other :wink: 
But still, taking some HTML/CSS/SCSS Tutorials will help you a lot :wink:

**Comment by w00fz** [comment]
Thanks for helping out @JoomFX, appreciate it!
@Cicorione you should consider joining our [Gitter Chat](https://gitter.im/gantry/gantry5), this kind of questions and help are more suited in there. Lot of people like Ivo and us from the Gantry team are constantly willing to help when and where we can.

Also the instant messaging works a lot better than the Github Issues!

**Comment by Cicorione** [comment]
Thanks to all for everything, I will use the Gitter Chat next time.

**Comment by JoomFX** [comment]
No problem at all, Djamil :wink:
